daemon/graphdriver/lcow/lcow_svm.go

```diff
@@ -0,0 +1,374 @@
+// +build windows
+
+package lcow
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+	"sync"
+	"time"
+
+	"io"
```

Comment:
Nit: Why is `io` separate from the previous set of imports?

Reply:
seems like a typo